teuthology/suite/util.py

```diff
@@ -254,7 +254,7 @@ def get_branch_info(project, branch, project_owner='ceph'):
         return resp.json()
 
 
-def package_version_for_hash(hash, kernel_flavor='basic', distro='rhel',
+def package_version_for_hash(hash, flavor='basic', distro='rhel',
```

Comment:
`basic` is not one of shaman flavors, should this default to `default` now?

Reply:
yep, agreed, will run some minimal testing in all major suites